ai-ml/trainium-inferentia/elastic-cache-redis.tf

```
@@ -0,0 +1,54 @@
+module "elasticache" {
+  count   = var.enable_rayserve_ha_elastic_cache_redis ? 1 : 0
```

all of our modules have a [`create`](https://github.com/terraform-aws-modules/terraform-aws-elasticache/blob/a483023ae9800c047d63206da7676afa79f1d82c/variables.tf#L1) variable to enable/disable resource creation. This is the correct way to toggle resource creation instead of using `count` on a module

```suggestion
  create   = var.enable_rayserve_ha_elastic_cache_redis
```